cmds/clients/contestcli/verbs.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/facebookincubator/contest/cmds/clients/contestcli/transport"
+	"github.com/facebookincubator/contest/pkg/api"
+	"github.com/facebookincubator/contest/pkg/config"
+	"github.com/facebookincubator/contest/pkg/jobmanager"
+	"github.com/facebookincubator/contest/pkg/types"
+
+	flag "github.com/spf13/pflag"
+)
+
+func run(requestor string, transport transport.Transport) error {
+	verb := strings.ToLower(flag.Arg(0))
+	if verb == "" {
+		fmt.Fprintf(flag.CommandLine.Output(), "Missing verb, see --help\n")
+		os.Exit(1)
+	}
+	var resp interface{}
+	var err error
+	switch verb {
+	case "start":
+		fmt.Fprintf(os.Stderr, "Reading from stdin...\n")
+		jobDesc, err := ioutil.ReadAll(os.Stdin)
+		if err != nil {
+			return fmt.Errorf("failed to read job descriptor: %v", err)
+		}
+
+		jobDescFormat := config.JobDescFormatJSON
+		if *flagYAML {
```

Comment:
given that any valid JSON is also valid YAML, why do we even need to make the distinction here?